.github/workflows/packages.yml

```diff
@@ -5,10 +5,14 @@ on:
     branches:
     - master
     paths:
-    - 'packages/**'
+    - 'main/**'
```

Comment:
As I said when you proposed it, I'm against renaming this directory because it truncates the git log for these directories.

Reply:
`packages, root-packages, x11-packages` also sounds better to me

Reply:
I've applied the changes as requested. I've also added a `repo.json` file which contains a map of directory containing packages and to the repository it belongs to. I've added it in order to not hard-code deb upload and artifact preparation logic for different repositories. Third part repositories who want to add their own new repo will benefit as they don't have to do look into the build scripts to fix them. Needs some testing especially for x11 and root packages and upload step.

I've till now only tested for main repo, packages seem to build well. Will look into building x11 and root packages. And for upload job I'll try running it on my fork after editing `scripts/aptly_api.sh` so that it only logs what it is doing and doesn't make any requests.